prog/vm/github_runner.rb

```diff
@@ -138,13 +129,17 @@ def before_run
       github_runner.update(runner_id: response[:runner][:id], ready_at: Time.now)
       # ./env.sh sets some variables for runner to run properly
       vm.sshable.cmd("./env.sh")
-      vm.sshable.cmd("common/bin/daemonizer 'sudo -u runner /home/runner/run.sh --jitconfig #{response[:encoded_jit_config].shellescape}' runner-script")
+
+      dir = "/home/runner"
+      command = "sudo -u runner #{dir}/run.sh --jitconfig #{response[:encoded_jit_config].shellescape}"
+      command = "/bin/bash -c '#{command} > #{dir}/#{SERVICE_NAME}.stdout 2> #{dir}/#{SERVICE_NAME}.stderr'"
+      vm.sshable.cmd("sudo systemd-run --working-directory #{dir} --unit #{SERVICE_NAME} --remain-after-exit #{command}")
```

Comment:
```suggestion
      vm.sshable.cmd("sudo systemd-run --working-directory #{dir} --unit #{SERVICE_NAME} --remain-after-exit -- #{command}")
```

Double-dash terminates option parsing, it's a good idea in settings that are automated and should bias towards being strict.

Reply:
I wasn't aware of the purpose of '--', this was enlightening for me. Thanks